mix.exs

```diff
@@ -68,7 +68,7 @@ defmodule Elixometer.Mixfile do
       {:dialyxir, "~> 0.5", only: :dev, runtime: false},
       {:ex_doc, "~> 0.18", only: :dev},
       {:excoveralls, "~> 0.8", only: [:dev, :test]},
-      {:pobox, "~>1.0.2"}
+      {:pobox, "~> 1.2.0"}
```

Comment:
Please use `{:pobox, "~>1.0"}` or `{:pobox, "~>1.2"}`.
Don't tie your project to a specific pobox version needlessly. Strict version dependency forces people like me to create (otherwise useless) forks :-)

Thanks
Peter

Reply:
I think this is @dependabot's doing. I'll fix it manually.

Reply:
👋 If you change this to `~> 1.2` Dependabot should keep that loose format when issuing updates in future.